tools/debug_tool/templates/pipeline_tree.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
```

Comment:
This file needs tidying up. I would removed the unused commented out regions and add more documentation to the code.